run_dir/design/ont_flowcell.html

```diff
@@ -0,0 +1,156 @@
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#tab_running_notes_content" role="tab" data-toggle="tab">Running Notes</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#tab_links_content" role="tab" data-toggle="tab">Links</a>
+        </li>
+      </ul>
+    </div>
+    <div class="tab-content">
+      <div class="tab-pane fade show active" id="tab_flowcell_information">
+        <h3>Barcode information</h3>
+        <table class="table table-bordered narrow-headers no-margin right_align_headers" id="ont_barcodes">
+            <thead id="ont_fc_table_head">
+                <tr class="sticky darkth">
+                    <th>Barcode</th>
+                    <th>Alias</th>
+                    <th>Read count passed</th>
+                    <th>% Read count passed total</th>
+                    <th>Bases passed (bp)</th>
+                    <th>% Bases passed total</th>
+                    <th>Average read length passed (bp)</th>
+                    <th>% Pass / Fail</th>
+                </tr>
+            </thead>
+            <tbody>
+            {% if barcodes %}
+              {% for bc in barcodes %}
+              <tr>
+                  <td class="bc_name">{{ barcodes[bc]["bc_name"] }}</td>
+                  <td class="bc_alias">{{ barcodes[bc]["barcode_alias"] }}</td>
+                  <td class="basecalled_pass_read_count">{{ '{:,}'.format(barcodes[bc]["basecalled_pass_read_count"]).replace(',', ' ') }}</td>
+                  <td class="basecalled_pass_read_count_pc">{{ barcodes[bc]["basecalled_pass_read_count_pc"] }}</td>
+                  <td class="basecalled_pass_bases">{{ '{:,}'.format(barcodes[bc]["basecalled_pass_bases"]).replace(',', ' ') }}</td>
+                  <td class="basecalled_pass_bases_pc">{{ barcodes[bc]["basecalled_pass_bases_pc"] }}</td>
+                  <td class="average_read_length_passed">{{ '{:,}'.format(barcodes[bc]["average_read_length_passed"]).replace(',', ' ') }}</td>
+                  <td class="accuracy">{{ barcodes[bc]["accuracy"] }}</td>
+              </tr>
+              {% end %} 
+            {% end %}
+            </tbody>
+        </table>
+      </div>
+      <div class="tab-pane fade" id="tab_running_notes_content">
+      </div>
+      <div class="tab-pane fade" id="tab_links_content">
+      </div>
+    </div>
+</div>
+
+<script src="/static/js/jquery.dataTables.min.js"></script>
+<!-- Table Sorting -->
+<script type="text/javascript">
+$( document ).ready(function() {
+  init_listjs();
+});
+// Initialize sorting and searching javascript plugin
+function init_listjs() {
+    // Setup - add a text input to each footer cell
+    $('#ont_barcodes tfoot th').each( function () {
+      var title = $('#ont_barcodes thead th').eq( $(this).index() ).text();
+      $(this).html( '<input size=10 type="text" placeholder="Search '+title+'" />' );
```

Comment:
Do we need a search bar here? 🤔

Reply:
Removed in commit 5b76e26f1ce6f8c6692d13ace994970aec936f4c